README.md

```diff
@@ -2,7 +2,11 @@
 
 [![npm version](https://badge.fury.io/js/rendition.svg)](http://badge.fury.io/js/rendition)
 [![Build Status](https://travis-ci.org/resin-io-modules/rendition.svg?branch=master)](https://travis-ci.org/resin-io-modules/rendition)
+[![Dependency Status](https://img.shields.io/david/resin-io-modules/rendition.svg)](https://david-dm.org/resin-io-modules/rendition)
+[![npm](https://img.shields.io/npm/dm/rendition.svg?maxAge=2592000)]()
+[![GitHub last commit](https://img.shields.io/github/last-commit/resin-io-modules/rendition.svg)]()
```

Comment:
Empty url :police_car: . How about pointing it to https://www.npmjs.com/package/rendition ?

Reply:
👍